docs/BR.md

```diff
@@ -532,19 +532,13 @@ This section defines the permitted processes and procedures for validating the A
 
 The CA SHALL confirm that prior to issuance, the CA has validated each Fully-Qualified Domain Name (FQDN) listed in the Certificate using at least one of the methods listed below.
 
-Completed validations of Applicant authority may be valid for the issuance of multiple Certificates over time. In all cases, the validation must have been initiated within the time period specified in the relevant requirement (such as Section 4.2.1 of this document) prior to Certificate issuance. For purposes of domain validation, the term Applicant includes the Applicant's Parent Company, Subsidiary Company, or Affiliate.
+Completed validations of Applicant's ownership or control of the domain under this section may be valid for the issuance of multiple Certificates over time, except that any completed validation of Applicant's ownership or control using a modified or deleted subsections may not be reused for future issuance of Certificates as of the effective date of the deletion or modification. In all cases, the validation must have been initiated within the time period specified in the relevant requirement (such as Section 4.2.1 of this document) prior to Certificate issuance. For purposes of domain validation, the term Applicant includes the Applicant's Parent Company, Subsidiary Company, or Affiliate. 
 
 CAs SHALL maintain a record of which domain validation method, including relevant BR version number, they used to validate every domain.
 
 Note: FQDNs may be listed in Subscriber Certificates using dNSNames in the subjectAltName extension or in Subordinate CA Certificates via dNSNames in permittedSubtrees within the Name Constraints extension.
 
-##### 3.2.2.4.1 Validating the Applicant as a Domain Contact
-
-Confirming the Applicant's control over the FQDN by validating the Applicant is the Domain Contact directly with the Domain Name Registrar. This method may only be used if:
-1.	The CA authenticates the Applicant's identity under BR Section 3.2.2.1 and the authority of the Applicant Representative under BR Section 3.2.5, OR
-2.	The CA authenticates the Applicant's identity under EV Guidelines Section 11.2 and the agency of the Certificate Approver under EV Guidelines Section 11.8; OR
-3.	The CA is also the Domain Name Registrar, or an Affiliate of the Registrar, of the Base Domain Name.
-Note: Once the FQDN has been validated using this method, the CA MAY also issue Certificates for other FQDNs that end with all the labels of the validated FQDN.  This method is suitable for validating Wildcard Domain Names.
+##### 3.2.2.4.1 Reserved
```

Comment:
So the challenge I see is that the document goes into effect immediately upon adoption (thus deleting this section), but your intent is clearly to permit this until March.

An alternative way to approach this is keep the existing text, but then add an:
`For certificates issued on or after March 1, 2018, this method SHALL NOT be used for validation, and completed validations using this method SHALL NOT be used for the issuance of certificates`.

as a new paragraph to both this and 3.2.2.4.5

and then the changes above

Reply:
I don't care about keeping it until March (and would be okay with an immediate deprecation date).  Considering there's two weeks of ballots plus an IPR review period, we'll be in March before it's effective anyway.  However, people may not vote for it without that language so I guess I better add it.

Reply:
Drat, that's a fair point with the explicit language - it can be problematic if it's retroactively enforced - e.g. if this didn't become adopted until March 8, what does it mean for those issued March 1 to March 8

My math shows this being (potentially) adopted mid-Feb, but if that's delayed, then I think your approach makes sense.

Reply:
The IPR review is a bit entertaining in this case.  I'm going to quick file a patent on not using bad validation methods ...